### ISSUE TITLE
Add support for only one polarization for Wavefront

### ISSUE THREAD

**Issue by buzmakov** [samoylv/WPG#71]
If we will keep only one polarization in wavefront, this may decrease memory usage and propagation time in 2 times

**Comment by samoylv** [comment]
yes, agreed. but SRW wavefront structure has 2 polarizations, how one can cope with that?

**Comment by CFGrote** [comment]
how about having an option "polarizated" or similar that tells srw to calculate only one polarization?

**Comment by samoylv** [comment]
1) well, seems we have one for s2e, Zoltan asked about that. but such type of modification in SRW could become an issue. nevertheless even on storage and reading/writing of the wavefronts one can also gain a lot. 

we should open a ticket in SRW as well - as soon as i remember there is no such an option.
